lightning/src/routing/scoring.rs

```diff
@@ -331,6 +331,20 @@ pub struct ProbabilisticScoringParameters {
 	/// Default value: 500 msat
 	pub base_penalty_msat: u64,
 
+	/// A fixed penalty in msats to apply to each channel, multiplied by the payment amount, in
+	/// excess of the [`base_penalty_msat`].
+	///
+	/// The purpose of the amount penalty is to avoid having fees dominate the channel cost (i.e.,
+	/// fees plus penalty) for large payments. The penalty is computed as the product of this
+	/// multiplier and `2^30`ths of the payment amount.
+	///
+	/// ie `amount_penalty_multiplier_msat * amount_msat / 2^30`
```

Comment:
```suggestion
	/// `base_penalty_amount_multiplier_msat * amount_msat / 2^30`
```

Reply:
Hm, am I mistaken to still think it should be `base_penalty_amount_multiplier_msat` here? Or am I misunderstanding/misreading the comment above?